ebos/eclwellmanager.hh

```diff
@@ -620,6 +620,42 @@ public:
     }
 
 
+    void
+    updateEclWell(int timeStepIdx, int well_index)
+    {
+        const auto& schedule = this->simulator_.vanguard().schedule();
+        const auto& wname = this->wells_[well_index]->name();
+        this->wells_[well_index] = std::make_shared<Well>( schedule.getWell(wname, timeStepIdx) );
```

Comment:
This does not look correct. Well from schedule is not
` typedef Opm::EclPeacemanWell<typeTag> Well;`

I quick fix is just to throw. And alternative is to add a method to the PeacemanWell that updated the eclWell. 

the `ebos_plain ` that uses this well model is probably not used by anybody and could IMO be removed.